### ISSUE TITLE
Option to set background image

### ISSUE THREAD

**Issue by blackbart420** [quodlibet/quodlibet#2584]
Hello,
sorry if this is not the right place to ask that but I do not know elsewhere.

Is it a way to configure a background image for the panel view ?
Else, is it an improvement you could consider to develop as an option ?

Thank you.

**Comment by lazka** [comment]
> Hello,
> sorry if this is not the right place to ask that but I do not know elsewhere.

this is the best place

> Is it a way to configure a background image for the panel view ?

No

> Else, is it an improvement you could consider to develop as an option ?

Is this something you can do with another player, and if yes, can you show how it looks there?

**Comment by blackbart420** [comment]
Thank you for answering me.

Unfortunelately no this is not a feature I use somewhere else...

At least, I wish I could customize the colors of the background and font for the 4 panels but I have no idea how difficult it would be to implement this feature.

![capture d ecran_2017-10-10_15-45-37](https://user-images.githubusercontent.com/15202659/31387294-8e5a5f30-add2-11e7-9fe2-19be2e31a925.png)

Thank you anyway, this player is awesome

**Comment by ZaxonXP** [comment]
Indeed, changing the colors of the player interface would be very nice. Dark colors would be less distractive when working on another screen (see Spotify player).
